src/io/github/thebusybiscuit/cscorelib2/protection/modules/PreciousStonesProtectionModule.java

```diff
@@ -22,6 +23,8 @@ public boolean hasPermission(OfflinePlayer p, Location l, Action action) {
 		if (!(p instanceof Player)) return false;
 		
         switch (action) {
+			case PVP:
+				return api.flagAppliesToPlayer((Player) p, FieldFlag.PREVENT_PVP, l);
```

Comment:
If the flag is called PREVENT_PVP shouldn't tjis be negated then?

Reply:
Right...